ds/data/processed/travel/travel_to_work_times_average.nuts2.yaml

```diff
@@ -0,0 +1,28 @@
+api_type: FETCH
+title: Average home to work travel time in minutes
+subtitle: Average home to work travel time where NUTS 2 region is home location (October to December)
+endpoint_url:

```

Comment:
no `description` here?

Reply:
No, I put a description in the LEP files to explain the chance of over-/under-estimation.

Reply:
Ah I see, what about duplicating it here so that it's visible in the app too?